test/runtests.jl

```diff
@@ -120,283 +120,313 @@ end
-    end
-    for solver in solvers_sos
-        jump_01_mixed(solver.opt)
-    end
-    for solver in solvers_sos
-        jump_01(solver.opt, solver.mode)#
-        jump_02(solver.opt, solver.mode)#
-        jump_03(solver.opt, solver.mode)
-        jump_04(solver.opt, solver.mode)
-        jump_05(solver.opt, solver.mode)#
-        jump_3SAT(solver.opt, solver.mode)
-        jump_06(solver.opt, solver.mode)#
-        jump_06_sv(solver.opt, solver.mode, CONFIG_4)
-        jump_07(solver.opt, solver.mode)#
-        jump_08(solver.opt, solver.mode)#
-        jump_09a(solver.opt, solver.mode) # fail on cbc positive SOS
-        jump_09b(solver.opt, solver.mode)
-        jump_11a(solver.opt, solver.mode)#
-        jump_11b(solver.opt, solver.mode)
-        jump_12(solver.opt, solver.mode)#
-        jump_14(solver.opt, solver.mode)
-        #
-        jump_16(solver.opt, solver.mode)
-    end
-    for solver in solvers_bin_exp
-        jump_01(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_01vec(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_02(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_03(solver.opt, solver.mode, CONFIG_3_hint)
-    end
-    for solver in solvers_fa
-        jump_01(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_01vec(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_02(solver.opt, solver.mode, CONFIG_3_hint)
-        jump_03(solver.opt, solver.mode, CONFIG_3_hint)
-    end
-    for solver in solvers_fa2
-        jump_01(solver.opt, solver.mode, CONFIG_3)
-        jump_01vec(solver.opt, solver.mode, CONFIG_3)
-        jump_02(solver.opt, solver.mode, CONFIG_3)
-        jump_03(solver.opt, solver.mode, CONFIG_3)
-    end
-    for solver in solvers_indicator
-        jump_01(solver.opt, solver.mode)
-        jump_01vec(solver.opt, solver.mode)
-        jump_02(solver.opt, solver.mode) # fail cbc - pass xpress 8.9
-        jump_03(solver.opt, solver.mode)
-    end
-    for solver in solvers_complements # requires KNITRO
-        # jump_01(solver.opt, solver.mode, CONFIG_3)
-        # jump_01vec(solver.opt, solver.mode, CONFIG_3)
-        # jump_02(solver.opt, solver.mode) # numerical isntability in ipopt
-        jump_03(solver.opt, solver.mode, CONFIG_3_start)
-        jump_03_vec(solver.opt, solver.mode, CONFIG_3_start)
-        jump_04(solver.opt, solver.mode, CONFIG_3_start)
-        # jump_05(solver.opt, solver.mode)
-        jump_3SAT(solver.opt, solver.mode)
-        jump_06(solver.opt, solver.mode)
-        jump_06_sv(solver.opt, solver.mode)
-        # jump_07(solver.opt, solver.mode, CONFIG_2)
-        jump_08(solver.opt, solver.mode, CONFIG_3_start)
-        jump_09a(solver.opt, solver.mode)
-        # jump_09b(solver.opt, solver.mode)
-        jump_11a(solver.opt, solver.mode)
-        jump_11b(solver.opt, solver.mode)
-        # jump_12(solver.opt, solver.mode)
-        # jump_14(solver.opt, solver.mode)
+    for consider_constrained_variables in [true, false]
```

Comment:
I dont think this should be run in all tests that would lead to run everything twice.
I think we should select a subset.

Reply:
See above